Code/Tools/FBuild/FBuildCore/WorkerPool/JobQueue.cpp

```diff
@@ -203,10 +203,22 @@ bool JobQueue::HaveWorkersStopped() const
 
 // GetNumDistributableJobsAvailable
 //------------------------------------------------------------------------------
-size_t JobQueue::GetNumDistributableJobsAvailable() const
+void JobQueue::GetNumDistributableJobsAvailable(
```

Comment:
The function is called in two places, and in one place, numJobsAvailable is not used, and in the other, it's checked redundantly. Additionally, I don't think there should ever be any nullptrs in the m_DistributableJobs_Available array.
Do you have any more information about this part of your change - maybe I've missed something?